integreat_cms/cms/views/events/event_list_view.py

```diff
@@ -202,6 +204,16 @@ def get(self, request, *args, **kwargs):
         )
         chunk = request.GET.get("page")
         event_chunk = paginator.get_page(chunk)
+
+        # DeepL available
+
+        if settings.DEEPL_ENABLED:
+            deepl = DeepLApi()
+            DEEPL_AVAILABLE = deepl.check_availability(request, language_slug)
+        else:
+            messages.error(request, _("Automatic translations are disabled"))
+            DEEPL_AVAILABLE = None
```

Comment:
```suggestion
            DEEPL_AVAILABLE = False
```
`None` works as well, but I think `False` is the more principled value for a boolean variable with two states.